### PR TITLE
Extending Wizard Spell Slots

### PR DESCRIPTION
**Pull request by Butterrobber202**
https://forums.aurorastation.org/viewtopic.php?f=18&t=10370
Done with this Suggestion here.

-Extended Druid, Cleric, Battlemage, Standard, and Nerco Spell Slot Count
-Gave battlemage blink so the poor guy has some way to get around.

### CODE REVIEW

File: code/modules/spells/spellbook/battlemage.dm [hunk 1]
```diff
@@ -31,4 +31,5 @@
 				/obj/item/weapon/melee/energy/wizard =					2,
 				/obj/item/weapon/monster_manual = 				    	2,
 				/obj/item/weapon/contract/apprentice = 					1
+				/spell/aoe_turf/blink = 					1,
```
Comment (Lohikar) [inline code/modules/spells/spellbook/battlemage.dm]: Bad comma.

File: code/modules/spells/spellbook/necromancer.dm [hunk 2]
```diff
@@ -10,7 +10,7 @@
 	title = "The Art of Necromancy"
 	title_desc = "Buy spells using your available spell slots. Artefacts may also be bought however their cost is permanent."
 	book_flags = CAN_MAKE_CONTRACTS
-	max_uses = 5
+	max_uses = 6
```
Comment (Alberyk) [inline code/modules/spells/spellbook/necromancer.dm]: No, don't buff necromancer.

Comment (Butterrobber202) [reply code/modules/spells/spellbook/necromancer.dm]: I thought it was nerfed?

Comment (OneOneThreeEight) [reply code/modules/spells/spellbook/necromancer.dm]: It was. Don't buff it.

**Comment by Arrow768** [comment]
Approved by mins

**Pull request merged**